app/models/host.rb

```diff
@@ -3,7 +3,7 @@
 # Host representation in insights compliance backend. Most of the times
 # these hosts will also show up in the insights-platform host inventory.
 class Host < ApplicationRecord
-  scoped_search on: %i[id name account_id]
+  scoped_search on: %i[id name account_id os_major os_minor]
```

Comment:
Let's strive for consistency with benchmarks and profiles:

```suggestion
  scoped_search on: %i[id name account_id os_major_release os_minor_release]
```

This is especially important for scoped_search because it makes it to our APIs.